package.json

```diff
@@ -44,7 +44,7 @@
     "lint": "yarn eslint --ext .ts .",
     "lint:fix": "yarn lint --fix",
     "lint:fix-cached": "yarn lint:fix --cache",
-    "check:circular": "yarn madge --circular ./packages/core",
+    "check:circular": "yarn madge --circular ./packages/generator",
```

Comment:
I just realised this command was just for core package.
I don't know whether it's easy to check multiple packages.
If not, maybe checking core would be more relevant.

Reply:
true, I just changed it to see if I solved the cyclic one you mentioned in generator. I change it back